lib/v4/migration-helpers/update-api-folder-structure.js

```diff
@@ -128,7 +93,7 @@ const updateApiFolderStructure = async (appPath) => {
   logger.info(
     `to revert: ${chalk.green('git')} reset HEAD --hard && ${chalk.green(
       'git'
-    )} cleanEmptyDirectories -fd`
+    )} clean -xdf`
```

Comment:
This may generate conflicts with [last week's PR](https://github.com/strapi/codemods/pull/7#discussion_r781995883)